CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### App Center Distribute
 
+* **[Fix]** Fix NullPointerExceptions occurring in Distribute module.
```

Comment:
Maybe make sense to clarify that this happens only when Dialog should be shown?

Reply:
If only "dialogs should be shown", as specifying exact cases would give too much unnecessary details. Thanks